.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+# Configures Dependabot to update package dependencies.
+version: 2
+updates:
+  # Check for version updates for Docker
```

Comment:
nit: docker image layers

Reply:
FYI you can edit the PR in your review :-)